Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -180,5 +182,80 @@
 
 assert(f5_error < tolerance)
 
+#==============================================
+# maxwell-boltzmann with constant bulk velocity
+#==============================================
+
+# load data
+bin_value_g, bin_data_g = read_reduced_diags_histogram("h6.txt")[2:]
+bin_value_uy, bin_data_uy = read_reduced_diags_histogram("h6uy.txt")[2:]
+
+# Expected values for beta and u = beta*gamma
+beta_const = 0.2
+g_const = 1. / np.sqrt(1 - beta_const * beta_const)
```

Comment:
Not needed anymore in Python3, but makes me feel warm and fuzzy:
```suggestion
g_const = 1. / np.sqrt(1. - beta_const * beta_const)
```
otherwise
```suggestion
g_const = 1 / np.sqrt(1 - beta_const * beta_const)
```